codefest/settings.py

```diff
@@ -167,16 +167,18 @@
 default_app = firebase_admin.initialize_app(cred)
 
 CORS_ORIGIN_WHITELIST = (
-    '*.codefest.tech',
-    '127.0.0.1:8080',
-    'localhost:8000',
-    '0.0.0.0:8080',
+    'https://*.codefest.tech',
+    'https://codefest.tech',
+    'http://http://127.0.0.1:8080',
```

Comment:
lol :rofl: 
Btw, no need of HTTP or https

Reply:
This was done by mistake :sweat_smile:
I didn't see this :sweat_smile: